azerothcore/azerothcore-wotlk
# fix(Scripts/Silithus): Stop banning people for using windstones

## Changes Proposed:
-  Remove the code that bans people if they don't have reagents.
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested in-game.
- 


## How to Test the Changes:


1. Go to the windstone without reagents
2. Use it & notice you are not banned
3.

## Known Issues and TODO List:


- [ ] Windstone script very hacky and needs to be fixed using spell scripts


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
